packages/synapse-react-client/package.json

```diff
@@ -32,7 +32,7 @@
     "url": "https://github.com/Sage-Bionetworks/synapse-web-monorepo/packages/synapse-react-client"
   },
   "dependencies": {
-    "@apidevtools/json-schema-ref-parser": "^9.1.2",
+    "@apidevtools/json-schema-ref-parser": "^14.1.1",
```

Comment:
Upgrade removes dependency on `@jsdevtools/ono`, so we can remove our patch for that package.